examples/correlation_alignment_example.py

```diff
@@ -87,20 +85,29 @@
             select_pathway(data.C.mean(indirect), signal_pathway)
         )
         data.setaxis("t2", lambda x: x - best_shift).register_axis({"t2": 0})
+        fl.image(data, ax=ax_list[3])
+        ax_list[3].set_title("Hermitian Test (t)")
         data.ft("t2")
-        data *= mysgn
-        fl.image(data, ax=ax_list[1])
-        ax_list[1].set_title("Phased and Centered")
-        #}}}
-        #{{{ Applying Correlation Routine to Align Data
+        fl.image(data, ax=ax_list[2])
+        ax_list[2].set_title("Hermitian Test (v)")
+        fig.tight_layout(rect=[0, 0.03, 1, 0.95])
```

Comment:
here I ended up rolling back some changes you had made to the images -- this seemed to be tied up with the whole sign-flip business.

Reply:
fixed